swap/swap.go

```diff
@@ -532,8 +532,8 @@ func (s *Swap) verifyContract(ctx context.Context, address common.Address) error
 	return contract.ValidateCode(ctx, s.backend, address)
 }
 
-// getContractOwner retrieve the owner of the chequebook at address from the blockchain
-func (s *Swap) getContractOwner(ctx context.Context, address common.Address) (common.Address, error) {
+// getContractIssuer retrieve the issuer of the chequebook at address from the blockchain
+func (s *Swap) getContractIssuer(ctx context.Context, address common.Address) (common.Address, error) {
```

Comment:
You could make it: `getIssuerAtContract`, or decide that we can cope with some ambiguity here.

Reply:
It makes more sense this way.